bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -593,6 +661,10 @@
 		<item-type>Location</item-type>
 		<label>Location</label>
 		<description>Location of the device</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>GeoLocation</tag>
+		</tags>
```

Comment:
The binding and the service does no do geolocation.
It is more a "Status" than a "Measurement".
It is a location provided by the user as a setting in the app.